charts/fluid/fluid/templates/crd-upgrade.yaml

```diff
@@ -0,0 +1,92 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: {{ .Release.Namespace }}
```

Comment:
Why do we need to put the script in configmap instead of docker image?

Reply:
Moved to `tools/crd-upgrade/upgrade-crds.sh`